.travis.yml

```diff
@@ -12,8 +12,6 @@ env:
   - TOXENV=django110
 
 matrix:
-  allow_failures:
```

Comment:
Tox has been running tests against python 3, it just hasn't cared about the failures until now.  If you look at [build #65](https://travis-ci.org/edx/web-fragments/builds/205032488) for this PR (current as of the latest version), you'll see that 65.4, 65.5, and 65.6 are run with Python 3. 

Compare that to [build #14](https://travis-ci.org/edx/web-fragments/builds/197391307) where the python 3 tests (14.6, 14.7, and 14.8) fail, but the build still passes.